GameLiftPlugin/Scripts~/windows/.export-release.ps1

```diff
@@ -0,0 +1,26 @@
+param(
+	[Parameter(Mandatory)]
+	[Alias("ServerSdk", "SdkVersion", "Sdk")]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $ServerSdkVersion,
+	[Parameter(Mandatory)]
+	[ValidatePattern("\d+\.\d+\.\d+")]
+	[string] $PluginVersion
+)
+
+$ROOT_DIR=Resolve-Path "$PSScriptRoot\..\..\.."
+$BUILD_DIR="$ROOT_DIR\GameLiftPlugin\build"
+$STAGING_PATH="$BUILD_DIR\staging-release"
+$README_PATH=Resolve-Path "$PSScriptRoot\..\PACKAGE_FOR_RELEASE.md"
```

Comment:
minor: IMO it's better to base this off the ROOT_DIR so that if this script moves only the root directory needs to be updated